docs/develop/api-reference/set-collections.md

```diff
@@ -117,4 +117,49 @@ Removes multiple elements from an existing set item.
 
 See [response objects](./response-objects.md) for specific information.
 
-</details>
\ No newline at end of file
+</details>
+
+### SetContainsElement
+Checks if a provided element is in the given set.
+
+| Name      | Type            | Description                                   |
+|-----------| --------------- | --------------------------------------------- |
+| cacheName | String          | Name of the cache.                            |
+| setName   | String          | Name of the set item to be altered.           |
```

Comment:
Since this operation does not modify the set, we should change the description to `Name of the set item`